theia-swift-docker/Dockerfile

```diff
@@ -98,4 +59,4 @@ ENV SHELL /bin/bash
 ENV USE_LOCAL_GIT true
```

Comment:
I think this is no longer required - we always use the local git now.

Reply:
@marcdumais-work sure